module/applications/actor/npc-sheet-2.mjs

```diff
@@ -0,0 +1,220 @@
+import ActorSheet5eNPC from "./npc-sheet.mjs";
+import ActorSheetV2Mixin from "./sheet-v2-mixin.mjs";
+import { simplifyBonus } from "../../utils.mjs";
+
+/**
+ * An Actor sheet for NPCs.
+ * @mixes ActorSheetV2
+ */
+export default class ActorSheet5eNPC2 extends ActorSheetV2Mixin(ActorSheet5eNPC) {
+  static get defaultOptions() {
+    return foundry.utils.mergeObject(super.defaultOptions, {
+      classes: ["dnd5e2", "sheet", "actor", "npc"],
+      width: 700,
+      height: 700,
+      resizable: true,
+      scrollY: [".sheet-body"],
+      tabs: [{ navSelector: ".tabs", contentSelector: ".tab-body", initial: "features" }]
+    });
+  }
+
+  /** @override */
+  static TABS = [
+    { tab: "features", label: "DND5E.Features", icon: "fas fa-list" },
+    { tab: "inventory", label: "DND5E.Inventory", svg: "backpack" },
+    { tab: "spells", label: "TYPES.Item.spellPl", icon: "fas fa-book" },
+    { tab: "effects", label: "DND5E.Effects", icon: "fas fa-bolt" },
+    { tab: "biography", label: "DND5E.Biography", icon: "fas fa-feather" }
+  ];
+
+  /* -------------------------------------------- */
+
+  get template() {
+    if ( !game.user.isGM && this.actor.limited ) return "systems/dnd5e/templates/actors/limited-sheet-2.hbs";
+    return "systems/dnd5e/templates/actors/npc-sheet-2.hbs";
+  }
+
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async _renderOuter() {
+    const html = await super._renderOuter();
+    const source = document.createElement("div");
+    source.classList.add("source-book");
+    source.innerHTML = `
+      <span></span>
+      <a class="config-button" data-action="source" data-tooltip="DND5E.SourceConfig"
+         aria-label="${game.i18n.localize("DND5E.SourceConfig")}">
+        <i class="fas fa-cog"></i>
+      </a>
+    `;
+    html[0].querySelector(".window-title")?.insertAdjacentElement("afterend", source);
+    source.querySelector(".config-button").addEventListener("click", this._onConfigMenu.bind(this));
+    return html;
+  }
+
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async _render(force=false, options={}) {
+    await super._render(force, options);
+    const [source] = this.element.find(".source-book");
+    if ( !source ) return;
+    const sourceEditable = this.isEditable && (this._mode === this.constructor.MODES.EDIT);
+    source.querySelector(".config-button")?.toggleAttribute("hidden", !sourceEditable);
+    source.querySelector(":scope > span").innerText = this.actor.system.details.source.label;
+  }
+
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async getData(options) {
+    const context = await super.getData(options);
+    const { attributes, details, resources } = this.actor.system;
+    context.encumbrance = attributes.encumbrance;
+
+    // CR
+    context.labels.cr = {
+      "0.125": "&frac18;",
+      "0.25": "&frac14;",
+      "0.5": "&frac12;"
+    }[details.cr] ?? details.cr;
```

Comment:
We have the `formatCR` utility method now, maybe we could even make that a handlebars helper so you don't have to worry about it in the sheet preparation at all.

```suggestion
    context.labels.cr = formatCR(details.cr);
```